### PR TITLE
Move config to prometheus-operator

### PR DESCRIPTION
**Pull request by mcbenjemaa**
towards https://github.com/giantswarm/giantswarm/issues/26221

This PR:

* remove the default config as it will be added in prometheus-operator

### Checklist

- [X] Update changelog in CHANGELOG.md.
- [X] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/observability-bundle/values.yaml [hunk 1]
```diff
@@ -68,20 +48,6 @@ apps:
               serviceMonitor:
                 https: true
```
Comment (QuentinBisson) [inline helm/observability-bundle/values.yaml:49]: Why keep the port and insecureSkipVerify? Also can you add them to proemtheus operator and link it here :D

File: helm/observability-bundle/values.yaml [hunk 2]
```diff
@@ -45,18 +45,12 @@ apps:
                 targetPort: 10257
                 selector:
                   component: kube-controller-manager
-              serviceMonitor:
-                https: true
-                insecureSkipVerify: true
             kubeScheduler:
               service:
                 port: 10259
```
[the remark this answers is not in the document]
Comment (mcbenjemaa) [reply helm/observability-bundle/values.yaml:50]: Sure

[the remark this answers is not in the document]
Comment (mcbenjemaa) [reply helm/observability-bundle/values.yaml]: Ready

[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/observability-bundle/values.yaml]: Did you test this PR?

[the remark this answers is not in the document]
Comment (mcbenjemaa) [reply helm/observability-bundle/values.yaml]: I need to test it, yes

**Comment by mcbenjemaa** [comment]
I can't test this anymore.

**Review by QuentinBisson — APPROVED** [review]

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**